❄️ Flaky test: story player navigation

Failure:
```
  1) story player navigation
       chrome
          Standalone environment 
            
             first story should be playing video:

      AssertionError: expected true to deeply equal false
      + expected - actual

      -true
      +false
      
      at _callee2$ (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:115:45)
      at tryCatch (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:61:40)
      at GeneratorFunctionPrototype.invoke [as _invoke] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:329:22)
      at GeneratorFunctionPrototype.prototype.<computed> [as next] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:94:21)
      at asyncGeneratorStep (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:3:103)
      at _next (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:5:194)
```
also
```
  1) story player navigation
       chrome
          Standalone environment 
            
             navigating to next story pauses the previous one:

      AssertionError: expected false to deeply equal true
      + expected - actual

      -false
      +true
      
      at _callee3$ (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:175:45)
      at tryCatch (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:61:40)
      at GeneratorFunctionPrototype.invoke [as _invoke] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:329:22)
      at GeneratorFunctionPrototype.prototype.<computed> [as next] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:94:21)
      at asyncGeneratorStep (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:3:103)
      at _next (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:5:194)
```

https://app.circleci.com/pipelines/github/ampproject/amphtml/5535/workflows/6d8d30c7-953d-40f9-8d61-d69d4357c7ee/jobs/75994
https://app.circleci.com/pipelines/github/ampproject/amphtml/5498/workflows/7135fc9b-b4bc-4c17-84ba-c48b6114e661/jobs/75341
https://app.circleci.com/pipelines/github/ampproject/amphtml/5498/workflows/7135fc9b-b4bc-4c17-84ba-c48b6114e661/jobs/75341

Skip PR incoming

@ampproject/wg-stories

--- a/examples/amp-story/video-one-page.html
+++ b/examples/amp-story/video-one-page.html
@@ -9,6 +9,8 @@
 
   <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
   <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/amp-viewer-integration-0.1.js"></script>
+  <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
 
 	<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
 	<style amp-custom>

--- a/examples/amp-story/video-one-page2.html
+++ b/examples/amp-story/video-one-page2.html
@@ -9,6 +9,8 @@
 
   <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
   <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/amp-viewer-integration-0.1.js"></script>
+  <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
 
 	<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
 	<style amp-custom>

--- a/extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js
+++ b/extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js
@@ -67,7 +67,7 @@ describes.endtoend(
         'paused'
       );
 
-      await timeout(500);
+      await timeout(800);
       await expect(isVideoPaused).to.eql(false);
     });
 
@@ -92,7 +92,7 @@ describes.endtoend(
         'paused'
       );
 
-      await timeout(500);
+      await timeout(800);
       await expect(isVideoPaused).to.eql(true);
     });
   }

--- a/test/e2e/test-amp-story-player-navigation.js
+++ b/test/e2e/test-amp-story-player-navigation.js
@@ -67,7 +67,7 @@ describes.endtoend(
         'paused'
       );
 
-      await timeout(500);
+      await timeout(800);
       await expect(isVideoPaused).to.eql(false);
     });
 
@@ -75,7 +75,7 @@ describes.endtoend(
       // Navigate to next story.
       await controller.click(player);
 
-      await timeout(500);
+      await timeout(800);
 
       const shadowHost = await controller.findElement(
         'div.i-amphtml-story-player-shadow-root-intermediary'
@@ -92,7 +92,7 @@ describes.endtoend(
         'paused'
       );
 
-      await timeout(500);
+      await timeout(800);
       await expect(isVideoPaused).to.eql(true);
     });
   }

